cards.js

```diff
@@ -0,0 +1,42 @@
+document.addEventListener("DOMContentLoaded" , () =>{
+    let button = document.querySelector("button")
+    const buildDeck = async () =>{
+        let display = document.querySelector("#display")
+        display.innerHTML=""
+        try {
+            let deck = await axios.get(`https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1`);
+            let deck_id = deck.data.deck_id
```

Comment:
DeckId should be kept outside this function's scope, it should be on the scope of the DOMContentLoaded event listener. This is because you want to keep track of this id after the function executes.